project/src/components/reviews/reviews-list.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReviewsListProps } from '../app/app-props';
+import ReviewsItem from './reviews-item';
+
+function ReviewsList(props: ReviewsListProps): JSX.Element {
+  return (
+    <ul className="reviews__list">
+      {props.reviews.map((review) => (
+        <ReviewsItem
+          id={review.id}
+          rating={review.rating}
+          comment={review.comment}
+          date={review.date}
+          user={review.user}
+          key={review.id}
+        />
```

Comment:
Чтобы меньше нажимать на кнопки можно так
```suggestion
      {props.reviews.map(({id, rating, comment, date, user}) => (
        <ReviewsItem
          id={id}
          rating={rating}
          comment={comment}
          date={date}
          user={user}
          key={id}
        />
```